src/___tests___/SearchResults.test.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("SearchResults component", () => {
+  const validProps = [
```

Comment:
This is an array of arrays, which isnt quite the best way to do this. Have a look at the comment in the component under test for a bit more explanation, your test should mirror the components props exactly